src/pages/office/components/seriousFaultComments/serious-fault-component.ts

```diff
@@ -0,0 +1,73 @@
+import { Observable } from 'rxjs/Observable';
+import { Component, OnInit, Input, ViewChild, ElementRef } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { StoreModel } from '../../../../shared/models/store.model';
+import { Store, select } from '@ngrx/store';
+import { getTests } from '../../../../modules/tests/tests.reducer';
+import { getCurrentTest } from '../../../../modules/tests/tests.selector';
+import { map } from 'rxjs/operators';
+import { getDrivingFaultComment } from '../../../debrief/debrief.selector';
+import { Subscription } from 'rxjs/Subscription';
+import { AddSeriousFaultComment } from '../../../../modules/tests/test_data/test-data.actions';
+import { InputSubscriptionActionDispatcher } from '../../../../shared/classes/input-area-action-dispatcher';
+
+interface SeriousFaultCommentComponentState {
+  competencyComment$: Observable<string>;
+}
+
+@Component({
+  selector: 'serious-fault-comment',
+  templateUrl: 'serious-fault-comment.html',
+})
+export class SeriousFaultCommentComponent extends InputSubscriptionActionDispatcher implements OnInit {
+  pageState: SeriousFaultCommentComponentState;
```

Comment:
Bit of a mismatch here, field name says `pageState`, type says `ComponentState`.